src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Slider/SliderHandler.iOS.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using UIKit;
+
+namespace Xamarin.Platform.Handlers
+{
+	public partial class SliderHandler : AbstractViewHandler<ISlider, UISlider>
+	{
+		internal UIColor? _defaultMinTrackColor;
+		internal UIColor? _defaultMaxTrackColor;
+		internal UIColor? _defaultThumbColor;
+
+		protected override UISlider CreateView()
+		{
+			var slider = new UISlider();
+
+			UpdateDefaultColors(slider);
+
+			slider.ValueChanged += OnControlValueChanged;
+
+			slider.AddTarget(OnTouchDownControlEvent, UIControlEvent.TouchDown);
+			slider.AddTarget(OnTouchUpControlEvent, UIControlEvent.TouchUpInside | UIControlEvent.TouchUpOutside);
+
+			return slider;
+		}
+
+		void UpdateDefaultColors(UISlider uISlider)
+		{
+			if (uISlider == null)
+				return;
+
+			_defaultMinTrackColor = uISlider.MinimumTrackTintColor;
```

Comment:
Let's move these to static properties that just get set once the first time the default property is pulled

Reply:
Should be properties or just static variables?